src/app/pages/HallOfFame/Main/Main.module.scss

```diff
@@ -13,12 +13,14 @@
       color: $orange;
       margin-bottom: 1rem;
     }
+
     &Issuer {
       font-size: 700;
     }
+
     &Button {
       margin-top: 2rem !important;
-      width: 13rem !important;
+      width: 14rem !important;
```

Comment:
Is it necesary to use !important? As far as I know it's not a good practice.

Reply:
Until some time ago, we were unable to work our styles along with Mui. I'll try to get rid of this and see if the styleProvider we include some weeks ago is working as expected and take the !important out.

Reply:
Done!